packages/lumx-core/src/scss/components/thumbnail/lumapps/_index.scss

```diff
@@ -43,18 +43,6 @@
         min-height: 1px;
         max-height: 100%;
 
```

Comment:
Is this class name `thumbnail__image` still used at all?

Reply:
Yes, in angularjs part